crypto/x509/v3_bitst.c

```diff
@@ -13,18 +13,6 @@
 #include <openssl/x509v3.h>
 #include "ext_dat.h"
 
-static BIT_STRING_BITNAME ns_cert_type_table[] = {
-    {0, "SSL Client", "client"},
-    {1, "SSL Server", "server"},
-    {2, "S/MIME", "email"},
-    {3, "Object Signing", "objsign"},
-    {4, "Unused", "reserved"},
-    {5, "SSL CA", "sslCA"},
-    {6, "S/MIME CA", "emailCA"},
-    {7, "Object Signing CA", "objCA"},
-    {-1, NULL, NULL}
-};
-
 static BIT_STRING_BITNAME key_usage_type_table[] = {
```

Comment:
`static const` while you are here

Reply:
can't, because of use in the macro below; added a comment.